tests/test_visualization.py

```diff
@@ -39,3 +39,13 @@ def test_staypoints_plot(self):
         stps.as_staypoints.plot(out_filename=tmp_file, radius=0.01, positionfixes=pfs, plot_osm=False)
         assert os.path.exists(tmp_file)
         os.remove(tmp_file)
+
+    def test_locations_plot(self):
+        tmp_file = 'tests/data/locations_plot.png'
```

Comment:
Use os.path module to construct operating system independent paths. I am actually surprised that this works on windows but for consistency you should construct the path using os.path anyways (as done in the other functions).